Nos/Views/Discover/FeaturedAuthorCategory.swift

```diff
@@ -9,27 +9,43 @@ enum FeaturedAuthorCategory: CaseIterable {
     case all
     /// A special type of category that includes all authors from the latest cohort.
     case new // swiftlint:disable:this identifier_name
-    case music
-    case news
-    case art // swiftlint:disable:this identifier_name
     case activists
-    case tech
-    case health
+    case art // swiftlint:disable:this identifier_name
+    case espanol
+    case food
     case gaming
+    case health
+    case lifestyle
+    case music
+    case news
+    case nzAustralia
+    case photography
+    case politics
+    case random
+    case sciFi
     case sports
+    case tech
 
     var text: LocalizedStringKey {
         switch self {
         case .all: "featuredAuthorCategoryAll"
         case .new: "featuredAuthorCategoryNew"
-        case .music: "featuredAuthorCategoryMusic"
-        case .news: "featuredAuthorCategoryNews"
-        case .art: "featuredAuthorCategoryArt"
         case .activists: "featuredAuthorCategoryActivists"
+        case .art: "featuredAuthorCategoryArt"
+        case .espanol: "featuredAuthorCategoryEspanol"
+        case .food: "featuredAuthorCategoryFood"
         case .gaming: "featuredAuthorCategoryGaming"
+        case .health: "featuredAuthorCategoryHealth"
+        case .lifestyle: "featuredAuthorCategoryLifestyle"
+        case .music: "featuredAuthorCategoryNews"
+        case .news: "featuredAuthorCategoryNews"
+        case .nzAustralia: "featuredAuthorCategoryNewzealand"
+        case .photography: "featuredAuthorCategoryPhotography"
+        case .politics: "featuredAuthorCategoryPolitics"
+        case .random: "featuredAuthorCategoryRandom"
+        case .sciFi: "featuredAuthorCategoryScifi"
```

Comment:
I'd probably match the capitalization here:
```suggestion
        case .sciFi: "featuredAuthorCategorySciFi"
```
though I know that also requires a Strings file update.